server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/bitpandapro/BitpandaProExchange.java

```diff
@@ -84,6 +86,23 @@ public static IExchangeAdvanced asExchange(String apikey, String preferredFiatCu
         CryptoCurrency.XRP.getCode()
     );
 
+    static private final Map<String, Integer> fiatPricePrecisions = new HashMap<>();
+    static private final Map<String, Integer> instrumentAmountPrecisions = new HashMap<>();
+
+    // https://api.exchange.bitpanda.com/public/v1/instruments
+    // https://api.exchange.bitpanda.com/public/v1/currencies
+    static {
+        fiatPricePrecisions.put("EUR", 2);
+        fiatPricePrecisions.put("CHF", 2);
```

Comment:
For prices the instrument's market precision is relevant, e.g. for XRP_EUR market precision is, allowing up to 5 decimal places on prices.

Reply:
This is addressed in #411, right?

Reply:
yes the fixed price calculation uses prices from the orderbook as is, which have an appropriate precision.